.internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml

```diff
@@ -11,12 +11,15 @@ data:
     #!/bin/bash
     set -e
 
-    YEAR=$(date '+%Y')
-    MONTH=$(date '+%m')
-    DAY=$(date '+%d')
-    HOUR=$(date '+%H')
-    MINUTE=$(date '+%M')
-    SECOND=$(date '+%S')
+    MDB_STAT="/usr/bin/mdb_stat"
+    EPOCH=`date '+%s'`
+
+    YEAR=$(date -d@$EPOCH '+%Y')
+    MONTH=$(date -d@$EPOCH '+%m')
+    DAY=$(date -d@$EPOCH '+%d')
+    HOUR=$(date -d@$EPOCH '+%H')
+    MINUTE=$(date -d@$EPOCH '+%M')
+    SECOND=$(date -d@$EPOCH '+%S')
```

Comment:
Do we need this script at all if we're using the new image.  Don't we have a copy in that repo?